File: src/algorithms/repo/gcd.c
#include <stdio.h>

int gcd(int a,int b) {
	if (!b) return a;
	return gcd(b,a%b);
}

int main(void) {

printf("%d\n",gcd(110,10));
printf("%d\n",gcd(1407,3441));
printf("%d\n",gcd(12,42));
printf("%d\n",gcd(3121,73165603));
printf("%d\n",gcd(3072,8388608));

return 0;
}

File: src/website/algorithms/repo/gcd.py
#!/usr/bin/python

def gcd(a,b):
	if b==0:
		return a
	return gcd(b,a%b)


print gcd(110,10)
print gcd(1407,3441)
print gcd(12,42)
print gcd(3121,73165603)
print gcd(3072,8388608)
